packages/composite-checkout/src/components/button.js

```diff
@@ -5,23 +5,33 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
 
+/**
+ * Internal Classes
+ */
+import joinClasses from '../lib/join-classes';
+
 export default function Button( {
+	className,
 	buttonState,
 	buttonType,
-	onClick,
-	className,
-	fullWidth,
+	isBusy,
 	children,
 	...props
 } ) {
+	const classNames = joinClasses( [
+		'checkout-button',
+		...( buttonState ? [ 'is-status-' + buttonState ] : [] ),
+		...( buttonType ? [ 'is-type-' + buttonType ] : [] ),
+		...( isBusy ? [ 'is-busy' ] : [] ),
+		...( className ? [ className ] : [] ),
+	] );
+
 	return (
 		<CallToAction
 			buttonState={ buttonState }
 			buttonType={ buttonType }
-			padding={ buttonState === 'text-button' ? '0' : '10px 15px' }
-			onClick={ onClick }
```

Comment:
Removed this and and `fullWidth` since they'll be automatically passed via `...props` and aren't needed in the class name logic.